src/routes/usergroups.ts

```diff
@@ -2,4 +2,377 @@ import * as express from "express";
+
+        if (!ugid || !user_id) {
+            return res.sendStatus(400);
+        }
+        else {
+            let mysql = `DELETE FROM UserGroupMembership 
+                WHERE user_id = '${user_id}' AND usergroup_id = '${ugid}'`;
+
+            db.connection.query(mysql, (error, results, fields) => {
+                if (error) {
+                    return res.sendStatus(500);
+                } 
+                else {
+                    return res.sendStatus(200);
+                }
+            });
+        }
+    });
+
+    /* /usergroup/:ugid/owners */
+    
+    // Get method (get all owners of an usergroup)
+    app.get('/api/v1/usergroup/:ugid/owners', (req: any, res) => {
+        let ugid = req.params.ugid;
+        if (!ugid) {
+            return res.sendStatus(400);
+        }
+        else {
+            let mysql = `SELECT * FROM UserGroupOwner 
+                WHERE usergroup_id = '${ugid}'`;
+            db.connection.query(mysql, (error, results, fields) => {
+                if (error) {
+                    return res.sendStatus(500);
+                }
+                else {
+                    return res.send(results);
+                }
+            });
+        }
+    });
+
+    // Post method (add new owner to the usergroup)
+    app.post('/api/v1/usergroup/:ugid/owners', (req: any, res) => {
+        let ugid = req.params.ugid;
+        let user_id = req.query.user_id;
+
+        if (!ugid || !user_id) {
+            return res.sendStatus(400);
+        }
+        else {
+            let mysql = `INSERT INTO UserGroupOwner VALUES('${user_id}', '${ugid}')`;
+            db.connection.query(mysql, (error, results, fields) => {
+                if (error) {
+                    return res.sendStatus(500);
+                }
+                else {
+                    return res.sendStatus(200);
+                }
+            });
+        }
+    });
+
+    // Delete Method (remove owner from an usergroup)
+    app.delete('/api/v1/usergroup/:ugid/owners/:uid', (req: any, res) => {
+        let ugid = req.params.ugid;
+        let user_id = req.params.uid;
+
+        if (!ugid || !user_id) {
+            return res.sendStatus(400);
+        }
+        else {
+            let mysql = `DELETE FROM UserGroupOwner 
+                WHERE user_id = '${user_id}' AND usergroup_id = '${ugid}'`;
+
+            db.connection.query(mysql, (error, results, fields) => {
+                if (error) {
+                    return res.sendStatus(500);
+                } 
+                else {
+                    return res.sendStatus(200);
+                }
+            });
+        }
+    });
+
+    /* /usergroup/:ugid/invited */
+
+    // Get method (get all invited users of an usergroup)
+    app.get('/api/v1/usergroup/:ugid/invited', (req: any, res) => {
+        let ugid = req.params.ugid;
+
+        if (!ugid) {
+            return res.sendStatus(400);
+        }
+        else {
```

Comment:
It's more readable if you remove the `else` here since you return inside the `if` anyways.
```suggestion
            if () {
              return something;
            }
            ... rest of code
```